src/plugins/urbanDictionary.ts

```diff
@@ -41,37 +41,51 @@ export default definePlugin({
             ],
             execute: async (args, ctx) => {
                 try {
-                    const { list: [definition] } = await (await fetch(`https://api.urbandictionary.com/v0/define?term=${args[0].value}`)).json();
+                    const query = encodeURIComponent(args[0].value);
+                    const { list } = await (await fetch(`https://api.urbandictionary.com/v0/define?term=${query}`)).json();
```

Comment:
just change this back then
```suggestion
                    const { list: [definition] } = await (await fetch(`https://api.urbandictionary.com/v0/define?term=${query}`)).json();
```